doc/source/getting_started/docker.rst

```diff
@@ -165,6 +212,67 @@ Verify your connection with:
     MAPDL Version:       RELEASE  2021 R1           BUILD 21.0
     PyMAPDL Version:     Version: 0.57.0
 
+
+You can also use :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>` to connect
+to an existing MAPDL instance by using the argument ``start_instance=False``:
+
+.. code:: python
+
+    from ansys.mapdl.core import launch_mapdl
+
+    mapdl = launch_mapdl(port=50053, start_instance=False) 
+
+
+Because of the linking between host ports and Docker ports (``-p`` argument),
+you do not need to specify the IP when connecting to a local container.
+However, if you are trying to connect to a WSL local distribution you need
+to specify the IP address of the WSL instance (normally ``127.0.0.1``) since
+no port mapping is done between both.
+For example:
```

Comment:
```suggestion
Because of the linking between host ports and Docker ports (``-p`` argument),
you do not need to specify the IP address when connecting to a local container.
However, if you are trying to connect to a WSL local distribution, you must
specify the IP address of the WSL instance (normally ``127.0.0.1``) because
no port mapping is done between both.
For example:
```